Subject: [PATCH] Consertos 10 dez
---
Index: docker-compose.ui.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/docker-compose.ui.yml b/docker-compose.ui.yml
--- a/docker-compose.ui.yml	(revision bc2f59d46cc0691d905e844a55581e6da716f3e1)
+++ b/docker-compose.ui.yml	(date 1763902913211)
@@ -19,7 +19,7 @@
     image: provectuslabs/kafka-ui:latest
     container_name: kafka-ui
     ports:
-      - "8082:8080"
+      - "8084:8080"
     depends_on:
       - kafka
     environment:
Index: src/main/java/com/ifba/sipapi/notification/application/NotificationConsumer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/ifba/sipapi/notification/application/NotificationConsumer.java b/src/main/java/com/ifba/sipapi/notification/application/NotificationConsumer.java
--- a/src/main/java/com/ifba/sipapi/notification/application/NotificationConsumer.java	(revision bc2f59d46cc0691d905e844a55581e6da716f3e1)
+++ b/src/main/java/com/ifba/sipapi/notification/application/NotificationConsumer.java	(date 1765372277371)
@@ -3,11 +3,14 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.ifba.sipapi.config.websocket.WebSocketBroadcaster;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.stereotype.Service;
 
 @Service
 @RequiredArgsConstructor
+@Log4j2
 public class NotificationConsumer {
 
     private final WebSocketBroadcaster broadcaster;
@@ -28,7 +31,7 @@
             }
 
         } catch (Exception ex) {
-            System.err.println("[Kafka->WS] error: " + ex.getMessage());
+            log.error("[Kafka->WS] error: " + ex.getMessage());
         }
     }
 }
Index: src/main/java/com/ifba/sipapi/config/websocket/NotificationHandler.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/ifba/sipapi/config/websocket/NotificationHandler.java b/src/main/java/com/ifba/sipapi/config/websocket/NotificationHandler.java
--- a/src/main/java/com/ifba/sipapi/config/websocket/NotificationHandler.java	(revision bc2f59d46cc0691d905e844a55581e6da716f3e1)
+++ b/src/main/java/com/ifba/sipapi/config/websocket/NotificationHandler.java	(date 1763899781424)
@@ -21,8 +21,6 @@
         var channel = (String) session.getAttributes().getOrDefault("channel", "common");
         log.info("WS connected id={} channel={}", session.getId(), channel);
         broadcaster.register(session);
-        try { session.sendMessage(new org.springframework.web.socket.TextMessage(
-                "{\"content\":\"connected to " + channel + "\"}")); } catch (Exception ignored) {}
         log.debug("[finish] NotificationHandler - afterConnectionEstablished");
     }
 
